src/templates/_includes/platform_selector.html

```diff
@@ -0,0 +1,16 @@
+<mkt-select class="compat-filter">
+  <mkt-selected><p>
+    {{ _('Show me apps for <mkt-selected-text>{selectedPlatform}</mkt-selected-text>')
```

Comment:
Do we need to support `<mkt-selected-text>` being within the translated string here? I feel like `Show me apps for` is the label before a select.